### ISSUE TITLE
More details required on Environment setup

### ISSUE THREAD

**Issue by globier** [microsoft/Partner-Smart-Office#62]
I'm setting up a new environment but the details in the wiki aren't quite clear as to the App Secrets and Tenant Identifiers.

Can you confirm what is required in these fields as by deploying via the deployment script I didn't  fill in the app secret so do I need to just create a new secret for the environment to use?
Also what is required in the Tenant Identifier?

Thanks

**Comment by alangreenwood** [comment]
Did you manage to resolve this? I have deployed the WebApp and added what I believe to be the correct secrets and keys from the Wiki articles, but it's not doing a data pull into the Cosmos Database?

Thanks, Alan.

**Comment by lukeapathy** [comment]
I have the same problem.  I can access the cosmos db from PowerBI, but not seeing any data.  I am not sure if I have the wrong environment details setup or if I am getting an error somewhere.

**Comment by lukeapathy** [comment]
I found my error.  I had to create a key in the Azure AD under Registrations and it worked

**Comment by alangreenwood** [comment]
Where did you find the error @lukeapathy ? Is there a log file I'm missing? I've followed the directions in the Wiki, created a new Key in Registrations and entered that into the Web App's "Add new Environment" Form.

Really disappointed in the documentation for this Web App. I don't suppose you've written up any clearer documentation as you've discovered how to use it?

Does the web app automatically call the functions? Do we have to run the functions ourselves? I'd appreciate some assistance.

**Comment by lukeapathy** [comment]
@alangreenwood I went to the app functions app (that other thing that was deployed in Web Apps from the template) and started clicking the run buttons.  1) I think that triggers the pull or push of data, 2) it has a little log.  After thinking a bit, I then went into my web app (the one hosting your environments) and generated a key for the web app and put it in the Azure portion of the environments config.  It then worked.  This step for me was very unclear and got lucky, but that did the trick.  I am configured for CSP.
